### DISCUSSION TITLE
[未解决]内网通过公网访问服务器失败

### DISCUSSION THREAD

**Discussion by jihanc** [immortalwrt/immortalwrt#842] [Q&A]
***我已经关闭Turbo ACC 网洛的所有功能以及重绑定保护。***  
OpenWrt设置：   

1. 我用OpenWrt配置了2个Lan口及一个Wan口，对应不同网段。Lan1口网段为10.0.0.0/8，网关IP为10.0.0.2。Lan2口网段为192.168.0.0/24，网关IP为192.168.0.1。我为每一个Lan口设置了不同防火墙区域。如图：Lan1口为CloudLan；Lan2口为lan![防火墙区域](https://user-images.githubusercontent.com/78247232/207006765-caf22324-635c-4458-a1cb-94e5d34af674.png)

2. Lan1口连接到服务器，Lan2口连接到Wifi。我有一个Minecraft服务器，服务器IP为10.0.0.3，内网开放端口为25565，通过OpenWrt映射到外网端口为：63481。 

3. 我的问题是：我通过Lan2口（192.168.0.0/24）使用服务器内网IP（10.0.0.3:25565）可以连接到服务器，Telnet我也进行过测试，是没有问题的。但是我通过公网IP（假设为：45.45.45.45）+端口号63481，无法连接到服务器，Telnet也无法连接。更换网络环境（使用流量），没有问题。  
**感谢解答我的疑问。**

**Comment by EnnawYang** [comment]
Wan口是不是还有设备，光猫？不是桥接？

**Comment by jihanc** [comment]
我的光猫是桥接模式，由openwrt进行拨号

**Comment by jihanc** [comment]
网络结构很简单，光猫>openwrt>服务器&WiFi。没有其他额外的网络设备

**Comment by 1715173329** [comment]
https://t.me/ctcgfw_openwrt_discuss/829940

**Reply by jihanc** [reply]
![image](https://user-images.githubusercontent.com/78247232/207194152-4d7e681a-86c3-4c99-8da8-ff7cde7f4012.png)
事实上并没有作用，问题还在。

**Comment by 1715173329** [comment]
Move to discussion.

**Comment by 52funny** [comment]
<img width="1264" alt="image" src="https://github.com/user-attachments/assets/fc933b56-9215-4861-bee4-85fb5d296a5a">
如果是通过域名访问不通过的话，可以把这个选项勾掉就可以访问了
